### PR TITLE
License

### LINKED ISSUE #450: Licensing
We might want to change our license from MIT to GPL or MPL or somthing (I'm not sure), a few of our dependencies use Apache, GPL, and MPL so I'm not sure.

### PR DESCRIPTION
**Pull request by Shuzhengz**
Adds MPL-2.0 license so we don't get sued lol
closes #450

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name = "autoclockspeed"
 version = "0.1.10"
 edition = "2018"
-license = "MIT"
+license = "MIT OR MPL-2.0"
```
Comment (JakeRoggenbuck) [inline Cargo.toml:5]: Do we know if crates.io supports this type of field?

**Comment by JakeRoggenbuck** [comment]
Thanks for the addition! LGTM

For legal reasons - I believe we are in legal good standing, as we use the MIT license. I do also think that having a MIT or MPL2.0 would be good, therefor we should add the MPL2.0.

File: Cargo.toml (hunk 1 above)
Comment (Shuzhengz) [reply Cargo.toml:5]: yep, i saw a few projects do that in their manifest

**Comment by Shuzhengz** [comment]
For legal resons, I agree with the above statement, with the exception of the case when such statement is false, then in this situatiuon I would legally not agree for legal reasons.

File: Cargo.toml (hunk 1 above)
Comment (JakeRoggenbuck) [reply Cargo.toml:5]: Sweet, thanks so much

**Review by JakeRoggenbuck — APPROVED** [review]
LGTM

**Comment by JakeRoggenbuck** [comment]
Thank you!

**Pull request merged**